### ISSUE TITLE
The scrubbing scrollbar UI hides the bottom line of the terminal -- often the most important line!

### ISSUE THREAD

**Issue by NHDaly** [asciinema/asciinema#397]
Most of the typing in any terminal session (once the scrollback has been filled) happens on the very bottom line. But in the Asciinema playback UI (which is otherwise **lovely** 😁❤️), the bottom line is covered whenever you mouse over the video!

This can make it quite difficult to scroll through a video and find the place you were looking for, because you can't see what the user is typing while you're scrolling!

This also makes it extremely difficult to copy/paste from the line that the user is about to enter into the terminal.

(For an example, here is an asciicast i recorded, where a lot of the action is happening on the bottom line, but you can't see it when the scrolling UI is displayed):
[![asciicast](https://asciinema.org/a/310469.svg)](https://asciinema.org/a/310469)

------------------------

Some ideas/proposals:
- Would it be possible to have the scrollback UI "push up" all the text by one line, so that you can still see the text on the bottom line? Rather than hovering up _over_ the text?
- Or could you just have the scrollback UI be always present? (I'm sure this is less desirable, aesthetically.)
- Or could the scrollback UI be somewhere else besides the bottom? Maybe the top? Or on the side? (The side might be too easily confused with an actual scrollback, which it of course isn't, thanks to things like ncurses.)

**Comment by NHDaly** [comment]
For completeness, here are two screenshots of the same frame with and without the mouse hovered over:
<img width="450" alt="Screen Shot Without Scroll UI" src="https://user-images.githubusercontent.com/1582097/77179792-56649c80-6a9f-11ea-93a4-27ab3f038eb9.png">
<img width="450" alt="Screen Shot With Scroll UI" src="https://user-images.githubusercontent.com/1582097/77179794-56fd3300-6a9f-11ea-89f1-94fac7559b27.png">

**Comment by sickill** [comment]
Yeah, that's fair. I guess one way of solving it would be to move the control bar below the whole terminal. Any other ideas?
